Sources/Utilities/KMCellConfiguration.swift

```diff
@@ -11,4 +11,9 @@ public enum KMCellConfiguration {
     
     /// if true then incoming messsage's sender name will be hidden on conversation
     public static var hideSenderName = false
+    
+    public static var staticTopMessage: String = ""
```

Comment:
Should it be added to CellConfig when it's at the conv VC level?

Reply:
created new file to have this configurations